packages/git.rb

```diff
@@ -37,6 +37,7 @@ class Git < Package
   depends_on 'musl_expat' => :build
 
   is_static
+  no_zstd
```

Comment:
Are we going to need to add this to every package that needs it now?

Reply:
It's really only needed for packages installed before zstd and a tar which supports zstd. So that's a very very small list.

And it may not be needed at all, since I think tar was built with zstd... I should check that...

Reply:
I need to add a check for zstd...

Reply:
So many compression options and so little time. :)